crates/subgraph/src/utils/mod.rs

```diff
@@ -1,5 +1,68 @@
+use alloy::primitives::utils::{format_units, parse_units, ParseUnits, Unit, UnitsError};
+
 mod order_id;
 mod slice_list;
 
 pub use order_id::*;
 pub use slice_list::*;
+
+pub const DAY: u64 = 60 * 60 * 24;
```

Comment:
using u64 for times is kinda dangerous because it's really easy for someone to mistake whether it is seconds, millis, nanos, etc.

why not just use the time functionality in rust?

Reply:
used rust `chrono::TimeDelta`